packages/project-access/src/project/cap.ts

```diff
@@ -146,15 +146,14 @@ export async function getCapModelAndServices(
 
     _logger?.info(`@sap-ux/project-access:getCapModelAndServices - Using 'cds.home': ${cds.home}`);
     _logger?.info(`@sap-ux/project-access:getCapModelAndServices - Using 'cds.version': ${cds.version}`);
-    _logger?.info(`@sap-ux/project-access:getCapModelAndServices - Using 'cds.root': ${cds.root}`);
 
     const capProjectPaths = await getCapCustomPaths(_projectRoot);
     const modelPaths = [
         join(_projectRoot, capProjectPaths.app),
         join(_projectRoot, capProjectPaths.srv),
         join(_projectRoot, capProjectPaths.db)
     ];
-    const model = await cds.load(modelPaths);
+    const model = await cds.load(modelPaths, { root: _projectRoot });
```

Comment:
is this supported only for recent cds versions? Do we need a check here.

Reply:
Good question. Need to check with Christian Georgi. @chgeo could you please comment?

Reply:
The `root` property has been there for a long time now.

Reply:
Thanks @chgeo